src/transformers/modeling_albert.py

```diff
@@ -749,7 +749,7 @@ def forward(
             Labels for computing the masked language modeling loss. Indices should be in ``[-100, 0, ...,
             config.vocab_size]`` (see ``input_ids`` docstring) Tokens with indices set to ``-100`` are ignored
             (masked), the loss is only computed for the tokens with labels in ``[0, ..., config.vocab_size]``
-        sentence_order_label (``torch.LongTensor`` of shape ``(batch_size,)``, `optional`):
```

Comment:
We only have two model types in the library that have a multi-loss model: 1) Some `ForPreTrainingModels` and the GPT `DoubleHeadModels`. I think it's cleaner if we force the loss names to be consistent here -> `sentence_order_label` is renamed to `next_sentence_label` as is done for `Bert`. The old argument is deprecated.

Reply:
Mmmm, this one has a different name than `next_sentence_label` because it is a different objective if my memory serves me right. So having a different name here does not seem like a bad idea.

Reply:
I think the SOP objective can still be represented by the `next_sentence_label`, as it can indicate whether the next sequence follows or precedes the first sequence.

However, `sentence_order_label` is a good name as it is, so why are we changing this? For consistency?

Reply:
Agree with you! Albert's Pretraining is slightly different to BERT's, so different names make sense here. Reverting the renaming.

Reply:
Thought it's the exact same objective, but it's not